pandas-stubs/core/indexes/accessors.pyi

```diff
@@ -464,6 +468,10 @@ class ArrayDescriptor:
         self, instance: IndexOpsMixin[Never], owner: type[IndexOpsMixin]
     ) -> ExtensionArray: ...
     @overload
+    def __get__(
+        self, instance: IndexOpsMixin[Int64Dtype], owner: type[IndexOpsMixin]
+    ) -> IntegerArray: ...
```

Comment:
I don't think this should exist because `Int64Dtype` is in `S1`, and I don't want to introduce that now, because it will open up a whole set of new issues.